kobo/apps/openrosa/libs/utils/logger_tools.py

```
@@ -1015,6 +1015,8 @@ def _get_instance_from_deprecated_id(
 
     try:
         instance = Instance.objects.get(uuid=old_uuid, xform_id=xform.pk)
+    except Instance.DoesNotExist:
+        raise InstanceIdMissingError(t('Invalid submission - deprecatedID not found.'))
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)